### PR TITLE
Update Trackmania.io

### PR DESCRIPTION
**Pull request by GreepTheSheep**
- Updated new link `leaderboard` -> `top`
- Removed startTimestamp

image

image

### CODE REVIEW

File: websites/T/Trackmania.io/presence.ts [hunk 1]
```diff
@@ -19,8 +19,7 @@ presence.on("UpdateData", async () => {
   const presenceData: PresenceData = {
     details: "An overview of the Trackmania Live Services.",
     largeImageKey: "logo",
-    state: `Unsupported page (${window.location.hash})`,
-    startTimestamp: Math.floor(Date.now() / 1000)
```
Comment (ririxidev) [inline websites/T/Trackmania.io/presence.ts:23]: why did you deleted it?

File: websites/T/Trackmania.io/dist/metadata.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
     "trackmania.io",
     "www.trackmania.io"
   ],
-  "version": "1.1.3",
+  "version": "1.2.0",
```
Comment (ririxidev) [inline websites/T/Trackmania.io/dist/metadata.json:12]:
```suggestion
  "version": "1.1.4",
```

**Review by rxri — CHANGES_REQUESTED** [review]

File: websites/T/Trackmania.io/presence.ts (hunk 1 above)
Comment (GreepTheSheep) [reply websites/T/Trackmania.io/presence.ts:23]: 1) this is useless
2) resets to 0 each time you change page

Comment (ririxidev) [reply websites/T/Trackmania.io/presence.ts:23]: That's how startTimestamp had to be, revert these changes.

**Review by rxri — APPROVED** [review]

**Review by Slowlife01 — APPROVED** [review]

**Pull request merged**